region/wreckedship/main/Wrecked Ship Energy Tank Room.json

```diff
@@ -373,6 +373,26 @@
       "requires": [],
       "note": "After teleporting, press down to retract Grapple, to prevent getting stuck inside the Chozo statue."
     },
+    {
+      "link": [1, 2],
+      "name": "Grapple Teleport Inside Wall",
+      "entranceCondition": {
+        "comeInWithGrappleTeleport": {
+          "blockPositions": [[3, 12], [3, 13]]
+        }
+      },
+      "requires": [
+        "canOffScreenMovement",
+        "Morph"
+      ],
+      "note": [
+        "After teleporting, Samus should be standing inside the wall.",
+        "Retract Grapple by pressing up, which will pull Samus down and right.",
```

Comment:
From top position I had to either end the grapple retract early and do a turn and roll out of the wall, or CF and xray climb.  I couldn't end the grapple in a way that let me spinjump.

Reply:
Interesting, tried it again and found that even from a save state after the teleport, sometimes after retracting and turning right it leaves me standing where I can spin jump, and other times it leaves me in a force crouch.

Looks like if after retracting I turn around left before turning back right, then it always leaves me standing. Added that to the note.